openpype/hosts/maya/plugins/load/load_multiverse_usd.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+import maya.cmds as cmds
+import maya.mel as mel
+
+from avalon import api
+
+from openpype.hosts.maya.api.lib import (
+    maintained_selection,
+    namespaced,
+    unique_namespace
+)
+from openpype.hosts.maya.api.pipeline import containerise
+
+
+class MultiverseUsdLoader(api.Loader):
+    """Load the USD by Multiverse"""
+
+    families = ["model", "usd", "usdComposition", "usdOverride"]
+    representations = ["usd", "usda", "usdc", "usdz", "abc"]
+
+    label = "Read USD by Multiverse"
+    order = -10
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name=None, namespace=None, options=None):
+
+        asset = context['asset']['name']
+        namespace = namespace or unique_namespace(
+            asset + "_",
+            prefix="_" if asset[0].isdigit() else "",
+            suffix="_",
+        )
+
+        # Create the shape
+        cmds.loadPlugin("MultiverseForMaya", quiet=True)
+
+        shape = None
+        transform = None
+        with maintained_selection():
+            cmds.namespace(addNamespace=namespace)
+            with namespaced(namespace, new=False):
+                import multiverse
+                shape = multiverse.CreateUsdCompound(self.fname)
+                transform = mel.eval('firstParentOf "{}"'.format(shape))
```

Comment:
Why are you resorting to mel for this? Seems like an odd choice.

```suggestion
                transform = cmds.listRelatives(shape, parent=True, fullPath=True)[0]
```

Reply:
Did in 73e691f379765841bf2169b477a62e5eceaa2100